Lib/subprocess.py

```diff
@@ -505,6 +511,11 @@ def run(*popenargs,
     return CompletedProcess(process.args, retcode, stdout, stderr)
 
 
+def _get_cleanup_timeout(timeout):  # For test injection.
+    if timeout is not None:
+        return timeout/20
```

Comment:
why timeout / 20? (instead of just returning original timeout)

Reply:
The original full timeout has already elapsed and expired, SIGKILL was
sent, we don't want to wait much longer.